server/reports/outbound-invoice-landscape-with-logo/2_3_0/src/template.html

```diff
@@ -15,7 +15,7 @@
 
     <tr> 
         <td class="header_section_field_left">{{ data.invoice.otherParty.address1 }}</td> 
-        <td class="header_section_field_right">{{t(k="report.confirmed-date", f="Confirmed date")}}: {{ data.invoice.createdDatetime | date(format="%d/%m/%Y") }}</td>
+        <td class="header_section_field_right">{{t(k="report.confirmed-date", f="Confirmed date")}}: {{ data.invoice.shippedDatetime | date(format="%d/%m/%Y") }}</td>
```

Comment:
Is `shipped date` correct for `confirmed date`?
I'm not actually 100% sure. Would prefer we called it shippedDate if so?

We also have verifiedDate, but maybe that doesn't apply for outbound?

Reply:
Ah it's picked for outbound. will fix